climada/engine/uncertainty/base.py

```diff
@@ -197,39 +199,39 @@ def __init__(self, unc_vars, samples=None, metrics=None,
         Parameters
         ----------
         unc_vars : dict
-            keys are names and values are climade.engine.uncertainty.UncVar 
+            keys are names and values are climade.engine.uncertainty.UncVar
         samples : pd.DataFrame, optional
             DataFrame of sampled parameter values. Column names must be
             parameter names (all labels) from all unc_vars.
             The default is pd.DataFrame().
         metrics : dict(), optional
-            Dictionnary of the CLIMADA metrics (outputs from 
+            Dictionnary of the CLIMADA metrics (outputs from
```

Comment:
minor point: "dictionary" is spelled with one n... also in few other places... please check

Reply:
O no, again the same mistake. Thanks!